lumen/ai/coordinator.py

```diff
@@ -661,21 +661,22 @@ async def _lookup_context(
             async for output in response:
                 if getattr(output, 'tables', None):
                     requested = [t for t in output.tables if t not in provided]
-                    loaded = '\n'.join([f'- {table}' for table in requested])
-                    istep.stream(f'Looking up schemas for following tables:\n\n{loaded}')
-                    table_info += await self._lookup_schemas(tables, requested, provided, cache=schemas)
-                if getattr(output, 'tools', None):
-                    for tool in output.tools:
-                        tool_messages = list(messages)
-                        if tool.instruction:
-                            mutate_user_message(
-                                f"-- Here are instructions of the context you are to provide: {tool.instruction!r}",
-                                tool_messages, suffix=True, wrap=True, inplace=False
-                            )
-                        response = await tools[tool.name].respond(tool_messages)
-                        if response is not None:
-                            istep.stream(f'{response}\n')
-                            tool_context += f'\n- {response}'
+                    loaded = '\n'.join(f'- {table}' for table in requested)
+                    if requested:
+                        istep.stream(f'Looking up schemas for following tables:\n\n{loaded}', replace=True)
+                        table_info += await self._lookup_schemas(tables, requested, provided, cache=schemas)
```

Comment:
The indentation here and below is still wrong. Will close and use https://github.com/holoviz/lumen/pull/1043/files instead.